app/Http/Controllers/FrontendStatusController.php

```diff
@@ -64,14 +64,14 @@ public function DeleteStatus(Request $request): JsonResponse|RedirectResponse {
 
     public function EditStatus(Request $request): JsonResponse|RedirectResponse {
         $this->validate($request, [
-            'body'          => 'max:280',
-            'businessCheck' => 'max:1',
+            'body'           => 'max:280',
+            'business_check' => 'digits_between:0,2'
```

Comment:
```suggestion
            'body'           => ['required', 'max:280'],
            'business_check' => ['required', 'digits_between:0,2'],
```
Would be nice if we define if it is required or nullable